packages/css-library/dist/tokens/scss/variables.scss

```diff
@@ -50,6 +51,16 @@ $color-blue-cool: #345d96;
 $color-blue-cool-light: #4a77b4;
 $color-blue-cool-lighter: #98afd2;
 $color-blue-cool-lightest: #e6e6e2;
+$color-hub-health-care: #3f57a6;
+$color-hub-education: #40807e;
+$color-hub-disability: #a23737;
+$color-hub-careers: #bd5727;
+$color-hub-pension: #4d8055;
+$color-hub-housing: #8e704f;
+$color-hub-life-insurance: #6f7a41;
+$color-hub-burials: #2378c3;
+$color-hub-family-member: #162e51;
+$color-hub-service-member: #162e51;
```

Comment:
Something I noticed when working on something else in the CSS Library is that there are these "hub" colors already existing for some reason:

https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/css-library/src/tokens/color.scss#L56-L68

Are they the same thing as what you're adding here? Is this entire file adding duplicate colors (on other words, do we even need that `src/tokens/color.scss` file at all)?

Reply:
Im not sure if that file is still being used, but I pushed an update to keep the hub colors it it in sync and use the scss variables.